src/app/passkey/[id]/page.tsx

```diff
@@ -35,19 +36,25 @@ export default function KeyPage({ params }: KeyPageProps) {
       <Header />
       <div className="flex flex-col items-center pt-10 min-h-screen p-4 bg-white text-center">
         <h1 className="text-2xl font-bold mb-4 text-primary-black">
-         {LABELS.passcode.title}
+          {LABELS.passcode.title}
         </h1>
         <div className="bg-alternate-green rounded-lg shadow-md text-black p-4">
           <span className="flex pb-4 text-lg">
             {tenant}
             <p className="pl-1">{LABELS.passcode.message}</p>
           </span>
 
-          <p>{LABELS.passcode.date} {date}</p>
-          <p>{LABELS.passcode.status} {status}</p>
+          <p>
```

Comment:
Are you formatting this with prettier or with something else? this formatting does not look like it came from prettier and I think it is causing an unnecessary change.